ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXRunner.scala

```diff
@@ -65,6 +66,7 @@ object SandboxOnXRunner {
           .acquire()
     }
 
+  @nowarn("cat=deprecation")
```

Comment:
Any way we can apply the `@nowarn` in a more precise way. This is a big method to completely remove deprecation warnings on.

Reply:
Yes, let me add it to each individual usage.